stuffListApi/public/controllers/controller.js

```diff
@@ -0,0 +1,176 @@
+var app = angular.module('myStuffApp', []);
+
+app.factory('Stuffs', ['$http', function($http) {
+    var stuffsFactory = {};
+
+    stuffsFactory.getStuffs = function() {
+        return $http.get('/stuffs');
+    }
+
+    stuffsFactory.createStuff = function(body) {
+        return $http.post('/stuffs', body);
+    }
+
+    stuffsFactory.readStuff = function(stuffId) {
+        return $http.get('/stuffs/' + stuffId);
+    }
+
+    stuffsFactory.updateStuff = function(stuffId, body) {
+        return $http.put('/stuffs/' + stuffId, body);
+    }
+
+    stuffsFactory.deleteStuff = function(stuffId) {
+        return $http.delete('/stuffs/' + stuffId);
+    }
+
+    stuffsFactory.deleteAllStuff = function() {
+        return $http.delete('/stuffs');
```

Comment:
Umm, I think it depends on the case, In this case yes, I think, because we're not handling any different kind of errors, I'd prefer to handle error or anything else where I need to because sometimes we need to handle different kind of error depending on the action.